src/templates/React InstantSearch/src/App.js.hbs

```diff
@@ -15,10 +15,7 @@ import {
 import PropTypes from 'prop-types';
 import './App.css';
 
-const searchClient = algoliasearch(
-  '{{appId}}',
```

Comment:
this was on purpose, because most api keys are quite a bit longer, and cross the 80 characters, not sure if it's worth changing if prettier will change it back :)

Reply:
So (very) long story short:

In local we run the E2E tests to build fake apps in `/var`, which doesn't give enough user access to run Prettier. On CircleCI, `/var` has enough rights to run Prettier.

So running the `test:e2e:templates` script didn't end up having the same snapshot output locally and on the CI.

I therefore decided to have the `hbs` template "prettified" by default because I was loosing too much time giving the correct access to my local machine (besides, we don't all use the same machines in the team).

Prettier is run when building apps and that the destination folder has the correct rights anyway so no worries!